src/sh/keptn/Keptn.groovy

```diff
@@ -414,8 +416,14 @@ def keptnAddStageResources(file, remoteUri) {
  * Stores the current local time in keptn.input.json
  */
 def markEvaluationStartTime() {
-    def startTime = getNow().toString()
-
+    //def startTime = getNow().toString()
+    
+    def LocalDateTime starttimelocal = LocalDateTime.now()       
+    def starttimeformatted = starttimelocal.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"))    
```

Comment:
@pcjeffmac the format is hard coded and gets used multiple times. can you put the format in a variable and perhaps create a reusable function for it? That way, in the future, we would only need to adapt in once if the format changes again.

Reply:
Yes, I have created a function to format the timestamp.  I will close this PR and create a new one.